etc/profile-a-l/firefox.profile

```diff
@@ -17,6 +17,7 @@ include globals.local
 noblacklist ${HOME}/.cache/mozilla
 noblacklist ${HOME}/.mozilla
 noblacklist ${RUNUSER}/*firefox*
+noblacklist ${RUNUSER}/psd/*firefox*
```

Comment:
`whitelist` it below.

Is there anything else in the PSD dir?

Reply:
is whitelist required ?

Reply:
Try it out ;)

If it should work, yes.